$NetBSD: patch-ae,v 1.2 2013/10/25 06:41:47 wiz Exp $

GNU Make check is broken -- does not recognize 4.0 as newer than 3.81.
http://developer.berlios.de/bugs/?func=detailbug&bug_id=19153&group_id=9034
Remove doxygen detection to avoid pulling it in unintendedly.

--- configure.orig	2013-01-27 16:51:06.000000000 +0000
+++ configure
@@ -45,18 +45,14 @@ if tibs.TARGET [0] == "windows":
     }
 
 # Check for tools
-tibs.check_program ("GNU Make", "make --version", ".*?([0-9\.]+).*", "3.81", True)
+#tibs.check_program ("GNU Make", "make --version", ".*?([0-9\.]+).*", "3.81", True)
 if tibs.check_program ("makedep", "makedep -V", ".*Version.*?([0-9\.]+).*", "0.1.0"):
     tibs.add_config_mak ("MAKEDEP", "makedep")
     makedep = "yes"
 else:
     makedep = "no"
 
-if tibs.check_program ("Doxygen", "doxygen --version", "([0-9\.]+)", "1.5.0"):
-    tibs.add_config_mak ("DOXYGEN", "doxygen")
-    doxygen = "yes"
-else:
-    doxygen = "no"
+doxygen = "no"
 
 tibs.pkgconfig_check_library ("glib-2.0", "2.0",
     "this is the low-level library that libLensFun relies on")
